rules/private/phases/phase_bootstrap_compile.bzl

```diff
@@ -55,7 +55,7 @@ def phase_bootstrap_compile(ctx, g):
             jar_creator = ctx.executable._jar_creator.path,
             main_class = main_class,
             output_jar = g.classpaths.jar.path,
-            scalacopts = " ".join(ctx.attr.scalacopts),
+            scalacopts = " ".join(ctx.attr.scalacopts + g.semanticdb.scalacopts),
```

Comment:
Do we need semanticdb for the bootstrap compiler? It's only used for a small number of internal targets.

Reply:
We don't. I didn't understand the full purpose of the bootstrap compiler when I wrote this. I'll fix it.